openapi-cli/src/test/java/io/ballerina/openapi/generators/client/BallerinaDiagnosticTests.java

```diff
@@ -86,7 +89,17 @@ public Object[][] singleFileProviderForDiagnosticCheck() {
                 {"xml_payload_with_ref.yaml"},
                 {"duplicated_response.yaml"},
                 {"complex_oneOf_schema.yaml"},
-                {"request_body_ref.yaml"}
+                {"request_body_ref.yaml"},
+                {"ballerinax_connector_tests/ably.yaml"},
+                {"ballerinax_connector_tests/azure.iot.yaml"},
+//                {"ballerinax_connector_tests/beezup.yaml"}, Disabled due to the issue openapi-tools/issues/1257
+                {"ballerinax_connector_tests/files.com.yaml"},
+                {"ballerinax_connector_tests/openweathermap.yaml"},
+                {"ballerinax_connector_tests/soundcloud.yaml"},
+                {"ballerinax_connector_tests/stripe.yaml"},
+//                {"ballerinax_connector_tests/vimeo.yaml"}, Disabled due to the issue openapi-tools/issues/1265
```

Comment:
Are these two issues gonna be blockers for generating connectors with 2201.4.1?

Reply:
Yes they are blockers. Issue https://github.com/ballerina-platform/openapi-tools/issues/1257 disrupts the generation of the connector. https://github.com/ballerina-platform/openapi-tools/issues/1265 generate connectors with compiler errors.